device/core/src/blob_upload/blob_upload_client.ts

```diff
@@ -124,4 +99,5 @@ export class BlobUploadClient implements BlobUpload {
       });
     }, done);
   }
+
```

Comment:
nit: There is no lint error from the blank line?

Reply:
there is no lint error on this but it isn't pretty. removed it.